python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -1776,3 +1776,16 @@ def _get_gradient(self,
                              fetch_list,
                              scope=scope,
                              return_numpy=False)))
+
+
+class OpTestTool:
+    @classmethod
+    def skipIf(cls, condition: object, reason: str):
```

Comment:
Please use the `snake case` name instead of the `camel case` name.

Reply:
OK